python/tk_nuke/menu_generation.py

```diff
@@ -44,19 +42,17 @@ def create_menu(self):
         """
         Render the entire Shotgun menu.
         """
-        # create main menu
-        nuke_menu = nuke.menu("Nuke")
-        menu_handle = nuke_menu.addMenu(self._menu_name) 
+        # create main Shotgun menu
+        menu_handle = nuke.menu("Nuke").addMenu(self._menu_name)
         
-        # the right click menu that is displayed when clicking on a pane 
-        self._pane_menu = nuke.menu("Pane") 
         # create tank side menu
-        node_menu_handle = nuke.menu("Nodes").addMenu(self._menu_name, icon=self.tank_logo)
+        node_menu_handle = nuke.menu("Nodes").addMenu(self._menu_name, icon=self._shotgun_logo)
```

Comment:
Shouldn't the shotgun logo resource be something provided by core and ideally setup by the engine?